exercises/sum-of-multiples/canonical-data.json

```diff
@@ -127,6 +127,15 @@
         "limit": 1
       },
       "expected": 0
+    },
+    {
+      "description": "solutions using include-exclude must extend to cardinality greater than 3",
+      "property": "sum",
+      "input": {
+        "factors": [2, 3, 5, 7, 11],
+        "limit": 10000
+      },
+      "expected": 39614537
     }
   ]
-}
\ No newline at end of file
+}
```

Comment:
as a matter of course I recommend that commits don't include unrelated changes such as whitespace

I just got bit by this when making my PR to sum-of-multiples though since my editor automatically adds the trailing newline on save. Perhaps this repo is needing an automatic formatter so that I don't have to expend routine objections like this. something like `jq .` would do.

Reply:
it doesn't matter to me whether this is changed, but I file the routine objection nevertheless